autoware_lanelet2_extension/lib/query.cpp

```diff
@@ -356,6 +356,18 @@ lanelet::ConstLineStrings3d getAllFences(const lanelet::LaneletMapConstPtr & lan
   return fences;
 }
 
+lanelet::ConstLineStrings3d getAllRoadBorders(const lanelet::LaneletMapConstPtr & lanelet_map_ptr)
```

Comment:
How about making a more general function ?
```suggestion
lanelet::ConstLineStrings3d getAllLinestringsWithType(const lanelet::LaneletMapConstPtr & lanelet_map_ptr, const std::string & type)
```

Reply:
Yeah, that actually is better. I created one [here](https://github.com/autowarefoundation/autoware_lanelet2_extension/pull/73/commits/4bb1fa0f8b8f57c13bdafa31417fbbe2f50a7184).

Reply:
I also removed the getAllRoadBorder function because it will just be a redundant function. using the general function is better imo.